boards/x86/intel_adl/intel_adl_crb.yaml

```diff
@@ -9,6 +9,7 @@ supported:
   - watchdog
   - pwm
   - gpio
+  - i2c
```

Comment:
TBH, it might be better if you group all of the additions to one PR. Or else you will be doing rebasing for all of these constantly.